conflictlog/memCachedConn.go

```diff
@@ -1,30 +1,38 @@
 package conflictlog
 
 import (
+	"encoding/binary"
+	"fmt"
+	"sync/atomic"
 	"time"
 
 	"github.com/couchbase/cbauth"
 	"github.com/couchbase/gomemcached"
 	mcc "github.com/couchbase/gomemcached/client"
 	"github.com/couchbase/goxdcr/base"
 	"github.com/couchbase/goxdcr/log"
+	"github.com/couchbase/goxdcr/metadata"
 	"github.com/couchbase/goxdcr/utils"
 )
 
-//var _ Connection = (*MemcachedConn)(nil)
+var _ Connection = (*MemcachedConn)(nil)
 
 type MemcachedConn struct {
-	logger     *log.CommonLogger
-	bucketName string
-	mccConn    *mcc.Client
+	logger        *log.CommonLogger
+	bucketName    string
+	mccConn       *mcc.Client
+	manifestCache *ManifestCache
+	opaque        uint32
 }
 
-func NewMemcachedConn(logger *log.CommonLogger, utilsObj utils.UtilsIface, bucketName string, addr string) (m *MemcachedConn, err error) {
+func NewMemcachedConn(logger *log.CommonLogger, utilsObj utils.UtilsIface, manCache *ManifestCache, bucketName string, addr string) (m *MemcachedConn, err error) {
 	user, passwd, err := cbauth.GetMemcachedServiceAuth(addr)
 	if err != nil {
 		return
 	}
 
+	logger.Debugf("memcached user=%s passwd=%s", user, passwd)
```

Comment:
Better remove for security sake

Reply:
yeah. removed. It was for troubleshooting.